script/deploy-rpms.pl

```diff
@@ -67,37 +65,40 @@
 	print STDERR "WARNING: $passfile does not exist!  New RPMs will not be signed!\n";
 }
 
-opendir(FILES, '.') or die "Unable to read current directory: $!";
-while (my $line = readdir(FILES)) {
+my $FILEDIR_HANDLE;
+opendir($FILEDIR_HANDLE, '<', '.') or die "Unable to read current directory: $!";
+while (my $line = readdir($FILEDIR_HANDLE)) {
 	next if ($line =~ /^\.\.?$/);
 	chomp($line);
 	if ($line =~ /^opennms-source-.*\.tar.(gz|bz2)$/) {
 		$FILE_SOURCE_TARBALL = $line;
 	} elsif ($line =~ /\.rpm$/) {
-		push(@FILES_RPMS, $line);
+		push(@$FILEDIR_HANDLE_RPMS, $line);
```

Comment:
I think this was mistakenly changed.

Reply:
I went ahead and pushed the fix for this in 09221ba03061f827f11745fc76aa389094aefa11.